src/table/body-cell/index.tsx

```diff
@@ -157,10 +156,12 @@ export function TableBodyCell<ItemType>({
 }: TableBodyCellProps<ItemType> & { isEditable: boolean }) {
   const editDisabledReason = rest.column.editConfig?.disabledReason?.(rest.item);
 
-  if (editDisabledReason && !rest.expandableProps) {
+  // Inline editing is deactivated for expandable column because editable cells are interactive
```

Comment:
should we put this in usage guidelines too? I know they're pretty complex already, but i can imagine this case to come up one day

Reply:
We plan to remove this constraint in the near future. It wasn't done as part of the initial release due to design and dev complexity.